aws/custom-microservices/README.md

```diff
@@ -0,0 +1,147 @@
+# AWS custom microservices deployment scripts
+
+This folder containing scripts and Kubernetes resources configurations to run ThingsBoard in Custom-Microservices mode on AWS cluster.
+
+## Prerequisites
+
+### Tools and roles configuration
+
+Please follow [this](aws/README.md) instructions to configure required tools.
+
+### Cluster configuration
+
+If you don't have an AWS cluster, you'll need to create it.
+In the `cluster.yml` file you can find suggested cluster configuration. 
+Here are the fields you can change depending on your needs:
+- `region` - should be the AWS region where you want your cluster to be located
+- `availabilityZones` - should specify the exact IDs of the region's availability zones
+- `instanceType` - the type of the instance with TB node (it's recommended to choose instance type with at least 4 CPU and 8 RAM)
+
+Command to create AWS cluster:
+```
+eksctl create cluster -f cluster.yml
```

Comment:
We recommend to use 4 CPU and 8 RAM but use t3.large. Please add a note that this will create 4 t3.large instances. You can change t3.large to m5.large or similar just to make sure users will not suffer from CPU rate limits.